plugin/plugin.pro

```diff
@@ -1,14 +1,21 @@
-TARGET=qofonoextdeclarative
+equals(QT_MAJOR_VERSION, 6) {
+    QOFONOEXT_SUFFIX = 6
+} else {
+    QOFONOEXT_SUFFIX =
+}
+
+
+TARGET=qofonoextdeclarative$${QOFONOEXT_SUFFIX}
```

Comment:
Let's not rename the libs

Reply:
Removed with push --force in 75fd66e9c0492215e70da4efdf7d06ab6e255239